.changelog/3635.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+helm: updated `server-statefulset.yaml` templating to handle custom Unix Domain Socket paths.
```

Comment:
**suggestion:** Can you mention something about this being for datadog ?

Reply:
Apologize for the delay - corrected and pushed. re-running tests now after main branch rebase 👍🏼